### PR TITLE
Use new feedids api instead of heavyweight feeds api

### PR DESCRIPTION
**Pull request by aasim**
**Task name**: multiple

**Description**: Updated the heavyweight feeds api to the lightweight feedids api.

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N) N

**Attached related issue:** (Y/N) https://dev.azure.com/mseng/AzureDevOps/_workitems/edit/1962856

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/DownloadPackageV1/task.json [hunk 1]
```diff
@@ -120,7 +120,7 @@
         {
             "target": "feed",
```
Comment (embetten) [inline Tasks/DownloadPackageV1/task.json:121]: version bump?

File: Tasks/TwineAuthenticateV1/task.json [hunk 2]
```diff
@@ -65,7 +65,7 @@
         {
             "target": "artifactFeed",
```
Comment (embetten) [inline Tasks/TwineAuthenticateV1/task.json:66]: version bump?

File: Tasks/DownloadPackageV1/task.json (hunk 1 above)
Comment (aasim) [reply Tasks/DownloadPackageV1/task.json:121]: 218 is not deployed yet right?

File: Tasks/DotNetCoreCLIV2/task.json [hunk 3]
```diff
@@ -501,14 +501,14 @@
         {
             "target": "feedRestore",
             "endpointId": "tfs:feed",
-            "endpointUrl": "{{endpoint.url}}/_apis/packaging/feeds?includeUrls=False",
+            "endpointUrl": "{{endpoint.url}}/_apis/packaging/feedids",
             "resultSelector": "jsonpath:$.value[*]",
             "resultTemplate": "{ \"Value\" : \"{{#if project}}{{{project.id}}}\\/{{/if}}{{{id}}}\", \"DisplayValue\" : \"{{{name}}}\" }"
```
Comment (jmyersmsft) [inline Tasks/DotNetCoreCLIV2/task.json:506]: These `resultTemplate`s also need to be updated (project is not a separate object in the new API)

File: Tasks/DownloadPackageV1/task.json (hunk 1 above)
Comment (embetten) [reply Tasks/DownloadPackageV1/task.json:121]: I thought it was needed - or the standared was - to bump the patch version - please correct me if I am wrong.

Comment (aasim) [reply Tasks/DownloadPackageV1/task.json:121]: I will bump it just to be safe

File: Tasks/DotNetCoreCLIV2/task.json (hunk 3 above)
Comment (aasim) [reply Tasks/DotNetCoreCLIV2/task.json]: fixed

**Review by jmyersmsft — APPROVED** [review]

**Pull request merged**